Description: Resolve differences between the tar and git for v1.2.0
 Debuild fails with a git checkout. Add the differences as patches.
 .
 sirikali (1.2.0-1) UNRELEASED; urgency=low
 .
   [Mhogo Mchungu]
   * New Features
      -- Add ability to add a favorite volume with mount options(external config file path,idle timeout,auto mount).
      -- Add a GUI option to select what secure key storage backend to use when auto mounting volumes.
Author: Mhogo Mchungu <mhogomchungu@gmail.com>

---

--- sirikali-1.2.0.orig/src/checkforupdates.cpp
+++ sirikali-1.2.0/src/checkforupdates.cpp
@@ -45,7 +45,14 @@ static void _show( QObject * obj,bool au
 
 		e += _tr( it ) + "\n\n" ;
 
-		show = ( it.at( 1 ) != it.at( 2 ) ) ;
+		const auto& a = it.at( 1 ) ;
+
+		const auto& b = it.at( 2 ) ;
+
+		if( a != "N/A" and b != "N/A" ){
+
+			show = ( a != b ) ;
+		}
 	}
 
 	if( autocheck ){
--- /dev/null
+++ sirikali-1.2.0/src/sirikali.pro
@@ -0,0 +1,63 @@
+
+SOURCES += lxqt_wallet/frontend/password_dialog.cpp \
+    lxqt_wallet/frontend/task.cpp \
+    lxqt_wallet/frontend/lxqt_wallet.cpp \
+    lxqt_wallet/frontend/lxqt_secret_service.cpp \
+    lxqt_wallet/frontend/lxqt_kwallet.cpp \
+    lxqt_wallet/frontend/lxqt_internal_wallet.cpp \
+    lxqt_wallet/frontend/changepassworddialog.cpp \
+    checkforupdates.cpp \
+    favorites.cpp \
+    oneinstance.cpp \
+    utility.cpp  \
+    sirikali.cpp  \
+    keydialog.cpp  \
+    plugin.cpp   \
+    walletconfig.cpp   \
+    siritask.cpp  \
+    main.cpp  \
+    readonlywarning.cpp \
+    walletconfiginput.cpp \
+    dialogmsg.cpp \
+    monitor_mountinfo.cpp \
+    tablewidget.cpp \
+    options.cpp
+
+HEADERS += checkforupdates.h \
+    favorites.h \
+    oneinstance.h \
+    utility.h  \
+    sirikali.h  \
+    keydialog.h  \
+    plugin.h   \
+    walletconfig.h   \
+    siritask.h  \
+    readonlywarning.h \
+    walletconfiginput.h \
+    dialogmsg.h \
+    monitor_mountinfo.h \
+    tablewidget.h \
+    options.h \
+    lxqt_wallet/backend/lxqtwallet.h \
+    lxqt_wallet/frontend/password_dialog.h \
+    lxqt_wallet/frontend/task.h \
+    lxqt_wallet/frontend/lxqt_wallet.h \
+    lxqt_wallet/frontend/lxqt_secret_service.h \
+    lxqt_wallet/frontend/lxqt_kwallet.h \
+    lxqt_wallet/frontend/lxqt_internal_wallet.h \
+    lxqt_wallet/frontend/changepassworddialog.h
+
+FORMS += sirikali.ui \
+    favorites.ui \
+    plugin.ui \
+    walletconfiginput.ui \
+    dialogmsg.ui \
+    keydialog.ui \
+    readonlywarning.ui \
+    walletconfig.ui \
+    options.ui \
+    lxqt_wallet/frontend/password_dialog.ui \
+    lxqt_wallet/frontend/changepassworddialog.ui
+
+TRANSLATIONS = ../translations/en_US.ts
+TRANSLATIONS = ../translations/fr_FR.ts
--- /dev/null
+++ sirikali-1.2.0/translations/en_US.ts
@@ -0,0 +1,1142 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1">
+<context>
+    <name>DialogMsg</name>
+    <message>
+        <location filename="../src/dialogmsg.ui" line="17"/>
+        <source>Dialog</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/dialogmsg.ui" line="29"/>
+        <source>&amp;Ok</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/dialogmsg.ui" line="42"/>
+        <source>&amp;Yes</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/dialogmsg.ui" line="55"/>
+        <source>&amp;No</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/dialogmsg.ui" line="68"/>
+        <source>text</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>LXQt::Wallet::changePassWordDialog</name>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="108"/>
+        <source>Create a new wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="110"/>
+        <source>Create</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="136"/>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="272"/>
+        <source>Passwords do not match</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="217"/>
+        <source>Wallet password could not be changed</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="226"/>
+        <source>Wallet could not be opened with the presented key</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.cpp" line="236"/>
+        <source>New passwords do not match</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>LXQt::Wallet::password_dialog</name>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.cpp" line="69"/>
+        <source>Wallet could not be opened with the presented key</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../src/checkforupdates.cpp" line="33"/>
+        <source>%1&quot;%2&quot; Installed Version Is : %3.
+Latest Version Is : %4.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/checkforupdates.cpp" line="78"/>
+        <location filename="../src/checkforupdates.cpp" line="121"/>
+        <source>Failed To Check For Updates.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/checkforupdates.cpp" line="79"/>
+        <location filename="../src/checkforupdates.cpp" line="123"/>
+        <source>ERROR</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/checkforupdates.cpp" line="105"/>
+        <source>Update Available</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/checkforupdates.cpp" line="119"/>
+        <source>Version Info</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="40"/>
+        <source>Kde Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="45"/>
+        <source>Internal Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="50"/>
+        <source>Gnome Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/utility.cpp" line="81"/>
+        <source>
+options:
+	-d   Path to where a volume to be auto unlocked/mounted is located.
+	-m   Tool to use to open a default file manager(default tool is xdg-open).
+	-e   Start the application without showing the GUI.
+	-b   A name of a backend to retrieve a password from when a volume is open from CLI.
+	     Supported backends are: &quot;internal&quot;,&quot;kwallet&quot; and &quot;gnomewallet.
+	     The first one is always present but the rest are compile time dependencies.
+	-k   When opening a volume from CLI,a value of &quot;rw&quot; will open the volume in read\write
+	     mode and a value of &quot;ro&quot; will open the volume in read only mode.
+	-z   Full path of the mount point to be used when the volume is opened from CLI.
+	     This option is optional.
+        -c   Set Configuration File Path when a volume is opened from CLI.
+	-i   Set inactivity timeout(in minutes) to dismount the volume when mounted from CLI.
+
+</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/utility.cpp" line="382"/>
+        <source>Manage Favorites</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/utility.cpp" line="383"/>
+        <source>Mount All</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/utility.cpp" line="410"/>
+        <source>about SiriKali</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/utility.cpp" line="600"/>
+        <source>hmac plugin.
+
+This plugin generates a key using below formular:
+
+key = hmac(sha256,passphrase,keyfile contents)</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>changePassWordDialog</name>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="14"/>
+        <source>Change Wallet&apos;s Password</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="42"/>
+        <source>C&amp;hange</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="55"/>
+        <source>&amp;Ok</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="126"/>
+        <source>Enter Current Password Below.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="145"/>
+        <source>Enter New Password Below.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="164"/>
+        <source>Re Enter New Password Below.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="183"/>
+        <source>&amp;Cancel</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="196"/>
+        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;An application &apos;&lt;span style=&quot; font-weight:600;&quot;&gt;%1&lt;/span&gt;&apos; has made a request for a password of its wallet &apos;&lt;span style=&quot; font-weight:600;&quot;&gt;%2&lt;/span&gt;&apos; to be changed&lt;/p&gt;
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/changepassworddialog.ui" line="214"/>
+        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Enter password information below to create a new wallet &apos;&lt;span style=&quot; font-weight:600;&quot;&gt;%1&lt;/span&gt;&apos; for application &apos;&lt;span style=&quot; font-weight:600;&quot;&gt;%2&lt;/span&gt;&apos;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>favorites</name>
+    <message>
+        <location filename="../src/favorites.ui" line="17"/>
+        <source>Manage Favorites</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="29"/>
+        <source>&amp;Add</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="64"/>
+        <source>Volume ID</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="72"/>
+        <source>Mount Point</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="89"/>
+        <source>Encrypted Folder Path</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="108"/>
+        <source>Mount Point Path</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="147"/>
+        <source>open file dialog</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.ui" line="163"/>
+        <source>&amp;Done</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="153"/>
+        <source>Remove Selected Entry</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="157"/>
+        <source>Cancel</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="208"/>
+        <location filename="../src/favorites.cpp" line="212"/>
+        <source>ERROR!</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="208"/>
+        <source>Encrypted folder address field is empty</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="212"/>
+        <source>Mount point path field is empty</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="246"/>
+        <source>Path To An Encrypted Folder</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/favorites.cpp" line="256"/>
+        <source>Path To Mount Folder</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>keyDialog</name>
+    <message>
+        <location filename="../src/keydialog.ui" line="29"/>
+        <source>&amp;Open</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="45"/>
+        <source>&amp;Cancel</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="58"/>
+        <location filename="../src/keydialog.cpp" line="196"/>
+        <location filename="../src/keydialog.cpp" line="761"/>
+        <location filename="../src/keydialog.cpp" line="836"/>
+        <source>Key</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="100"/>
+        <location filename="../src/keydialog.cpp" line="85"/>
+        <source>Volume Name</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="142"/>
+        <source>Mount In &amp;Read Only Mode</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="155"/>
+        <source>&amp;Share Mount Point</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="168"/>
+        <source>O&amp;ptions</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.ui" line="201"/>
+        <source>Volume Path</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="83"/>
+        <source>&amp;Create</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="106"/>
+        <source>Unlocking &quot;%1&quot;</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="110"/>
+        <source>Mount Path</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="197"/>
+        <location filename="../src/keydialog.cpp" line="765"/>
+        <location filename="../src/keydialog.cpp" line="801"/>
+        <source>KeyFile</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="198"/>
+        <location filename="../src/keydialog.cpp" line="773"/>
+        <source>HMAC+KeyFile</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="199"/>
+        <location filename="../src/keydialog.cpp" line="769"/>
+        <location filename="../src/keydialog.cpp" line="815"/>
+        <source>Key+KeyFile</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="237"/>
+        <source>Create A New &quot;%1&quot; Volume</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="273"/>
+        <source>Passphrase Quality: 0%</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="275"/>
+        <source>Passphrase Quality: %1%</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="280"/>
+        <source>Passphrase Quality: 100%</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="327"/>
+        <location filename="../src/keydialog.cpp" line="350"/>
+        <source>Select A Folder To Create A Mount Point In.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="421"/>
+        <location filename="../src/keydialog.cpp" line="433"/>
+        <source>Select A File To Be Used As A Keyfile.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="456"/>
+        <location filename="../src/keydialog.cpp" line="463"/>
+        <location filename="../src/keydialog.cpp" line="514"/>
+        <location filename="../src/keydialog.cpp" line="543"/>
+        <location filename="../src/keydialog.cpp" line="548"/>
+        <location filename="../src/keydialog.cpp" line="553"/>
+        <location filename="../src/keydialog.cpp" line="558"/>
+        <location filename="../src/keydialog.cpp" line="563"/>
+        <location filename="../src/keydialog.cpp" line="568"/>
+        <location filename="../src/keydialog.cpp" line="573"/>
+        <location filename="../src/keydialog.cpp" line="578"/>
+        <location filename="../src/keydialog.cpp" line="583"/>
+        <location filename="../src/keydialog.cpp" line="588"/>
+        <location filename="../src/keydialog.cpp" line="593"/>
+        <location filename="../src/keydialog.cpp" line="596"/>
+        <location filename="../src/keydialog.cpp" line="613"/>
+        <location filename="../src/keydialog.cpp" line="624"/>
+        <location filename="../src/keydialog.cpp" line="633"/>
+        <location filename="../src/keydialog.cpp" line="666"/>
+        <location filename="../src/keydialog.cpp" line="675"/>
+        <location filename="../src/keydialog.cpp" line="684"/>
+        <source>ERROR</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="456"/>
+        <source>Volume Name Field Is Empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="463"/>
+        <source>Key Field Is Empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="501"/>
+        <source>ERROR!</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="501"/>
+        <source>Internal Wallet Is Not Configured.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="514"/>
+        <source>The Volume Does Not Appear To Have An Entry In The Wallet.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="543"/>
+        <source>Failed To Unlock A Cryfs Volume.
+Wrong Password Entered.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="548"/>
+        <source>Failed To Unlock An Encfs Volume.
+Wrong Password Entered.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="553"/>
+        <source>Failed To Unlock A Gocryptfs Volume.
+Wrong Password Entered.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="558"/>
+        <source>Failed To Unlock A Securefs Volume.
+Wrong Password Entered.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="563"/>
+        <source>Failed To Complete The Request.
+Cryfs Executable Could Not Be Found.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="568"/>
+        <source>Failed To Complete The Request.
+Encfs Executable Could Not Be Found.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="573"/>
+        <source>Failed To Complete The Request.
+Gocryptfs Executable Could Not Be Found.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="578"/>
+        <source>Failed To Complete The Request.
+Securefs Executable Could Not Be Found.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="583"/>
+        <source>Failed To Create Mount Point.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="588"/>
+        <source>Failed To Unlock The Volume.
+Not Supported Volume Encountered.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="593"/>
+        <source>Failed To Complete The Task.
+Backend Not Responding.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="596"/>
+        <source>Failed To Complete The Task.
+An Unknown Error Has Occured.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="613"/>
+        <source>Encrypted Folder Path Is Already Taken.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="624"/>
+        <location filename="../src/keydialog.cpp" line="666"/>
+        <source>Mount Point Path Already Taken.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="633"/>
+        <location filename="../src/keydialog.cpp" line="684"/>
+        <source>Atleast One Required Field Is Empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="675"/>
+        <source>Encrypted Folder Appear To Not Be Present.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="739"/>
+        <source>WARNING</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="739"/>
+        <source>KeyFile Contents Will Be Trancated On The First Encountered NewLine Character.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="813"/>
+        <location filename="../src/keydialog.cpp" line="825"/>
+        <source>Plugin name</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/keydialog.cpp" line="848"/>
+        <source>Keyfile path</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>oneinstance</name>
+    <message>
+        <location filename="../src/oneinstance.cpp" line="108"/>
+        <source>Previous instance seem to have crashed,trying to clean up before starting</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/oneinstance.cpp" line="115"/>
+        <source>There seem to be another instance running,exiting this one</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>options</name>
+    <message>
+        <location filename="../src/options.ui" line="17"/>
+        <source>Set Mount Options</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/options.ui" line="38"/>
+        <source>&amp;OK</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/options.ui" line="93"/>
+        <source>Unlock A Cryfs/Gocryptfs/Securefs Volume With Specified Configuration File.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/options.ui" line="122"/>
+        <source>Automatically Unmount After Specified Minutes of Inactivity Is Reached.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/options.cpp" line="53"/>
+        <source>Select Cryfs/Gocryptfs Configuration File</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/options.cpp" line="82"/>
+        <source>ERROR</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/options.cpp" line="82"/>
+        <source>Idle Time Field Requires Digits Only If Not Empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>password_dialog</name>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="14"/>
+        <source>lxqt wallet service</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="42"/>
+        <source>The application &apos;%1&apos; has requested to open the wallet &apos;%2&apos;.Enter the password below for this wallet to unlock it</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="74"/>
+        <source>Wallet &apos;%1&apos; does not exist,do you want to create it?</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="90"/>
+        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Wallet &lt;span style=&quot; font-weight:600;&quot;&gt;&apos;%1&apos;&lt;/span&gt; does not exist, do you want to create it?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="107"/>
+        <source>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;An application &lt;span style=&quot; font-weight:600;&quot;&gt;&apos;%1&apos;&lt;/span&gt; has requested to open a wallet&lt;span style=&quot; font-weight:600;&quot;&gt; &apos;%2&apos;&lt;/span&gt;. Enter the password below for this wallet to unlock it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="124"/>
+        <source>&amp;Unlock</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="141"/>
+        <source>&amp;Cancel</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="154"/>
+        <source>ok</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/lxqt_wallet/frontend/password_dialog.ui" line="167"/>
+        <source>&amp;Ok</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>plugin</name>
+    <message>
+        <location filename="../src/plugin.ui" line="17"/>
+        <source>Key Generator Using A Passphrase And A KeyFile</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.ui" line="29"/>
+        <source>&amp;Set Key</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.ui" line="42"/>
+        <source>&amp;Cancel</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.ui" line="93"/>
+        <source>Create an encryption key that is made up of a passphrase and a keyfile.
+
+A volume created with a key generated here should be opened with &quot;hmac&quot; plugin.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.ui" line="115"/>
+        <source>Passphrase</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.ui" line="144"/>
+        <location filename="../src/plugin.cpp" line="138"/>
+        <source>KeyFile</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.cpp" line="102"/>
+        <location filename="../src/plugin.cpp" line="127"/>
+        <source>ERROR</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.cpp" line="102"/>
+        <source>KeyFile Not Set</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/plugin.cpp" line="127"/>
+        <source>Failed To Generate Key</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>readOnlyWarning</name>
+    <message>
+        <location filename="../src/readonlywarning.ui" line="14"/>
+        <source>WARNING</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/readonlywarning.ui" line="26"/>
+        <source>Do Not Show This Message Again.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/readonlywarning.ui" line="39"/>
+        <source>Setting This Option Will Cause The Volume To Open In Read Only Mode.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/readonlywarning.ui" line="58"/>
+        <source>&amp;Ok</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>sirikali</name>
+    <message>
+        <location filename="../src/sirikali.ui" line="17"/>
+        <source>SiriKali</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="55"/>
+        <source>Volume Path</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="63"/>
+        <source>Mount Point Path</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="71"/>
+        <source>File System</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="79"/>
+        <source>Mode</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="95"/>
+        <source>&amp;Create Volume</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="108"/>
+        <source>Mount &amp;Volume</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="115"/>
+        <source>&amp;Refresh</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.ui" line="122"/>
+        <source>&amp;Menu</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="229"/>
+        <source>Auto Open Mount Point</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="232"/>
+        <source>Reuse Mount Point</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="235"/>
+        <source>Autocheck For Updates</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="238"/>
+        <source>Set Mount Point Prefix</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="241"/>
+        <source>Unmount All</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="245"/>
+        <location filename="../src/sirikali.cpp" line="270"/>
+        <source>Internal Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="247"/>
+        <source>Change Password</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="257"/>
+        <source>Key Storage</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="271"/>
+        <location filename="../src/sirikali.cpp" line="371"/>
+        <source>KDE Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="272"/>
+        <location filename="../src/sirikali.cpp" line="375"/>
+        <source>Gnome Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="277"/>
+        <source>Favorites</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="281"/>
+        <source>Select Language</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="284"/>
+        <source>Check For Updates</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="287"/>
+        <source>About</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="289"/>
+        <source>Quit</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="299"/>
+        <source>Show/Hide</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="429"/>
+        <source>Manage Favorites</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="433"/>
+        <source>Mount All</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="578"/>
+        <source>ERROR: Volume Path Not Given.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="616"/>
+        <source>ERROR: Key Not Found In The Backend.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="640"/>
+        <source>ERROR: Failed To Unlock Requested Volume.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="645"/>
+        <source>ERROR: Failed To Unlock Requested Backend.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="675"/>
+        <location filename="../src/sirikali.cpp" line="988"/>
+        <location filename="../src/sirikali.cpp" line="1099"/>
+        <source>ERROR</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="675"/>
+        <source>Failed To Read Volume Properties</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="679"/>
+        <source>INFORMATION</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="718"/>
+        <source>Block Size: %1</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="719"/>
+        <source>Used Blocks: %2</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="720"/>
+        <source>Free Blocks: %3</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="721"/>
+        <source>Total Blocks %4</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="722"/>
+        <source>Used Space: %5</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="723"/>
+        <source>Free Space: %6</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="724"/>
+        <source>Total Space: %7</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="725"/>
+        <source>Used %: %8</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="798"/>
+        <source>Unmount</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="800"/>
+        <source>Open Folder</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="802"/>
+        <source>Properties</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="818"/>
+        <source>Close Menu</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="870"/>
+        <source>WARNING</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="871"/>
+        <source>Could Not Open Mount Point Because &quot;%1&quot; Tool Does Not Appear To Be Working Correctly.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="989"/>
+        <source>Permission To Access The Volume Was Denied
+Or
+The Volume Is Not Supported</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="1009"/>
+        <source>Select An Encrypted Volume Directory</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/sirikali.cpp" line="1099"/>
+        <source>Failed To Unmount %1 Volume</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>walletconfig</name>
+    <message>
+        <location filename="../src/walletconfig.ui" line="17"/>
+        <source>Manage Volumes In A Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.ui" line="32"/>
+        <source>&amp;Add</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.ui" line="51"/>
+        <source>&amp;Delete</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.ui" line="64"/>
+        <source>Do&amp;ne</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.ui" line="111"/>
+        <source>Volume ID</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.ui" line="116"/>
+        <source>Comment</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.cpp" line="78"/>
+        <source>Delete Entry</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.cpp" line="93"/>
+        <source>WARNING!</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.cpp" line="93"/>
+        <source>Are you sure you want to delete a volume with an id of &quot;%1&quot;?</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.cpp" line="177"/>
+        <source>ERROR!</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfig.cpp" line="177"/>
+        <source>Failed To Add the Key In The Wallet.</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>walletconfiginput</name>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="14"/>
+        <source>Add An Entry To Wallet</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="26"/>
+        <source>&amp;Add</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="39"/>
+        <source>&amp;Cancel</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="61"/>
+        <source>Volume ID</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="80"/>
+        <source>Comment</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="99"/>
+        <source>Key</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.ui" line="118"/>
+        <source>Repeat Key</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.cpp" line="95"/>
+        <location filename="../src/walletconfiginput.cpp" line="104"/>
+        <source>ERROR!</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.cpp" line="95"/>
+        <source>Atleast one required field is empty</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.cpp" line="104"/>
+        <source>Passphrases do not match</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../src/walletconfiginput.cpp" line="127"/>
+        <source>Select A Volume</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+</TS>
